Glyssen/ProjectExporter.cs

```diff
@@ -357,18 +358,34 @@ private void GenerateExcelFile(string path, IEnumerable<List<object>> data)
 				sheet.Column(columnNum++).AutoFit(2d, sheet.DefaultColWidth); // verse
 				sheet.Column(columnNum++).AutoFit(2d, 20d); // character ID
 
-				// add a column for the localized character id
```

Comment:
I don't think we want to remove this code. If we really don't want to auto-fit this column (and why not?), it doesn't feel like it has anything to do with this Jira issue. In any case, I think we would need to increment columnNum.